CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
+## [2.1.1]
```

Comment:
Lets put "Unreleased" in place of the date so it's clearer if this goes into master but we don't have time to do a release. Also, I think the changes are big enough that we go for 2.2.0.

Reply:
OK